### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by Adimote**
currently the tests are failing, this fixes that

### CODE REVIEW

File: tests/test_camera.py [hunk 1]
```diff
@@ -1,76 +0,0 @@
-import os
```
Comment (RealOrangeOne) [inline tests/test_camera.py]: Did these camera tests still pass? If so i'd suggest leaving them in, unless you want to remove _all_ the camera api with it

Comment (Adimote) [inline tests/test_camera.py]: they didn't pass

File: robot/robot.py [hunk 2]
```diff
@@ -167,7 +174,14 @@ def mode(self):
...
         return self._game.mode
 
-    def __del__(self):
+    def close(self):
         # stop the polling threads
         for camera in self.known_cameras:
             camera._stop_poll()
+        for board_type in self.all_known_boards:
+            for board in board_type:
+                board._clean_up()
```
Comment (RealOrangeOne) [inline robot/robot.py]: I'd suggest using `del board` here. So custom deconstructors can be used on each board if necessary

**Comment by RealOrangeOne** [comment]
Use images from `sb-vision`. The image locations are known, and will provide a better and more standard test

**Pull request merged**